app/models/component/manifest/app_container.rb

```diff
@@ -34,7 +34,6 @@ def app_resources
       def environment_settings
         env = settings.except('PORT')
                       .map { |k, v| { name: k.to_s, value: v.to_s }.with_indifferent_access }
-        env << { name: '_JAVA_OPTIONS', value: '-Xmx1280m -Xms1280m' }.with_indifferent_access
```

Comment:
The only concern what most deployed Java apps will default to once this is removed?